src/main/java/sg/ncl/testbed_interface/PasswordResetForm.java

```diff
@@ -16,6 +16,8 @@
     private String errMsg;
 
     public boolean isPasswordOk() {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) The Cyclomatic Complexity of this method "isPasswordOk" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)